src/wrap/legendary.rs

```diff
@@ -0,0 +1,47 @@
+use itertools::Itertools;
+
+use crate::{resource::config::RootsConfig, scan::heroic::get_legendary_installed_games};
+
+use super::LaunchParser;
+
+/// Deserialization of legendary game metadata from calling 'legendary list-installed --json'
+#[derive(Debug, serde::Deserialize)]
+pub struct LegendaryGameInfo {
```

Comment:
It doesn't look like this is used. Are we reusing the existing Heroic config support?

Reply:
It isn't anymore since I reused `scan::heroic::get_legendary_installed_games` -> removed.